ci-operator/step-registry/insights-qe/iqe-cost-management-plugin/insights-qe-iqe-cost-management-plugin-commands.sh

```diff
@@ -0,0 +1,136 @@
+
+# create secret for pulling quay image
+username="$(cat ${SECRETS_DIR}/username)"
+password="$(cat ${SECRETS_DIR}/password)"
+oc create secret docker-registry cspi-pull-secret --docker-server=quay.io \
+--docker-username=${username} --docker-password=${password}
+
+# create secret for vault access
+oc apply -f $SECRETS_DIR/insights-vault-pull
+
+cat <<EOF | oc create -f -
+apiVersion: v1
+kind: Pod
+metadata:
+  name: insights-pod
+spec:
+  restartPolicy: Never
+  securityContext:
+    seccompProfile:
+      type: RuntimeDefault
+  containers:
+  - name: insights-container
+    image: quay.io/cloudservices/iqe-tests:cost-management
+    command: ["/bin/sh", "-c"]
+    args: ["iqe tests plugin cost_management -m cost_interop -vv --junitxml=test_run.xml"]
+    env:
+    - name: KUBECONFIG
+      value: "/kubeconfig/kubeconfig"
+    - name: IQE_MARKER_EXPRESSION
+      value: core
+    - name: IQE_FILTER_EXPRESSION
+    - name: IQE_REQUIREMENTS
+    - name: IQE_REQUIREMENTS_PRIORITY
+    - name: IQE_TEST_IMPORTANCE
+    - name: IQE_PLUGINS
+      value: cost-management
+    - name: IQE_LOG_LEVEL
+      value: INFO
+    - name: IQE_PARALLEL_ENABLED
+      value: "false"
+    - name: IQE_PARALLEL_WORKER_COUNT
+      value: "2"
+    - name: IQE_TESTS_LOCAL_CONF_PATH
+      value: "None"
+    - name: ENV_FOR_DYNACONF
+      value: prod
+    - name: DYNACONF_IBUTSU_URL
+      value: https://ibutsu-api.apps.ocp4.prod.psi.redhat.com/
+    - name: DYNACONF_IQE_VAULT_LOADER_ENABLED
+      value: "true"
+    - name: DYNACONF_IQE_VAULT_MOUNT_POINT
+      valueFrom:
+        secretKeyRef:
+          key: mountPoint
+          name: iqe-vault
+    - name: DYNACONF_IQE_VAULT_URL
+      valueFrom:
+        secretKeyRef:
+          key: url
+          name: iqe-vault
+    - name: DYNACONF_IQE_VAULT_ROLE_ID
+      valueFrom:
+        secretKeyRef:
+          key: roleId
+          name: iqe-vault
+    - name: DYNACONF_IQE_VAULT_SECRET_ID
+      valueFrom:
+        secretKeyRef:
+          key: secretId
+          name: iqe-vault
+    imagePullPolicy: Always
+    resources:
+      limits:
+        cpu: 500m
+        memory: 1000Mi
+      requests:
+        cpu: 200m
+        memory: 500Mi
+    securityContext:
+        allowPrivilegeEscalation: false
+        runAsNonRoot: true
+        capabilities:
+          drop:
+            - ALL
+    volumeMounts:
+    - name: kubeconfig
+      mountPath: /kubeconfig
+  volumes:
+  - name: kubeconfig
+    secret:
+      secretName: kubeconfig-secret
+  imagePullSecrets:
+  - name: cspi-pull-secret
+EOF
+
+oc wait --for=condition=ContainersReady=true pod/insights-pod --timeout=200s || true
+oc get pod insights-pod
+
+echo "Waiting for test results..."
+
+while true; do
+  oc exec insights-pod -- ls . > $SHARED_DIR/Files
+  if ! grep -q 'test_run.xml' $SHARED_DIR/Files ; then
+    sleep 5
+  else
+    break
+  fi
+done
+
+echo "Retrieve test results..."
+oc cp insights-pod:test_run.xml "${ARTIFACT_DIR}/test_run.xml"
```

Comment:
```suggestion
oc cp insights-pod:test_run.xml "${ARTIFACT_DIR}/junit_test_run.xml"
```

To report the XML on the testgrid dashboard the name should start with `junit_`